checker/src/main/java/org/checkerframework/checker/calledmethods/DescribeImages.astub

```diff
@@ -0,0 +1,27 @@
+package com.amazonaws.services.ec2;
+
+import org.checkerframework.checker.calledmethods.qual.*;
+
+// Interface
+class AmazonEC2 {
+    DescribeImagesResult describeImages(
+    // any combination of withX/setX has to be permitted if an owner has been set or an imageId has been set or an executable user list has been set
```

Comment:
I don't understand this comment.  Is it trying to say, "The receiver must have its owner, imageId, or executable user set."?

Reply:
Yes, but it is also saying that any other combination of methods is permitted. I'll rewrite to make it more clear.